math/catalan/catalannumber_test.go

```diff
@@ -0,0 +1,40 @@
+// catalannumber_test.go
+// description: Test for returns the Catalan number
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see catalannumber.go
+
+package catalan
+
+import "testing"
+
+func TestCatalanNumber(t *testing.T) {
+	type args struct {
+		n int
+	}
+	tests := []struct {
+		name string
+		args args
+		want int
```

Comment:
There is only one argument right? Why did you make a struct to contain n when you can just have this?

```suggestion
	tests := []struct {
		name string
		arg  int
		want int
```

Reply:
Done